### ISSUE TITLE
npmPackage.graphql-request

### ISSUE THREAD

**Issue by toastal** [NixOS/nixpkgs#127523]
**Project description**
[`graphql-request`](https://github.com/prisma-labs/graphql-request) is a "minimal GraphQL client supporting Node and browsers for scripts or simple apps". More and more services are offering GraphQL in tandem or replacing REST. Escaping for Bash+cURL and JSON is pretty hard to follow, and I think scripts with a `node` shebang would be a lot easier.

An example from their docs

```javascript
#!/usr/bin/env node
import { request, gql } from 'graphql-request'

const query = gql`
  {
    Movie(title: "Inception") {
      releaseDate
      actors {
        name
      }
    }
  }
`

request('https://api.graph.cool/simple/v1/movies', query).then((data) => console.log(data))
```

**Metadata**

* homepage URL: https://github.com/prisma-labs/graphql-request
* source URL: https://github.com/prisma-labs/graphql-request
* license: MIT
* platforms: Any that are Node-compatible (though version 14.x is required)

**Comment by tomodachi94** [comment]
Per the JavaScript/NodeJS [package inclusion information](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/javascript.section.md#javascript-packages-inside-nixpkgs-javascript-packages-nixpkgs):

> As a rule of thumb, the package set should only provide end user software packages, such as command-line utilities. Libraries should only be added to the package set if there is a non-npm package that requires it.

I don't think this provides a CLI.

**Comment by tomodachi94** [comment]
Closing as wontfix, please reopen if community consensus about NPM packages changes.
